addons/remix-tree-button/addon.json

```diff
@@ -21,7 +25,42 @@
       "matches": ["projects"]
     }
   ],
+  "settings": [
+    {
+      "name": "Link placement",
+      "id": "type",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "new",
+          "name": "Add button next to \"Copy Link\""
+        },
+        {
+          "id": "replace",
+          "name": "Replace \"Remixes \u2192 View all\" link"
+        }
+      ],
+      "default": "new"
+    },
+    {
+      "name": "Open remix tree by clicking remixes icon next to @heart.svg@star.svg",
```

Comment:
I think this should be one of the options above instead of its own toggle

Reply:
This would be a good opportunity to allow multiple of these to be turned on at the same time. Maybe I should split everything into toggles so you don't have to lose one button to add another.